src/Disks/IO/ReadBufferFromRemoteFSGather.cpp

```diff
@@ -86,12 +86,15 @@ SeekableReadBufferPtr ReadBufferFromHDFSGather::createImplementationBuffer(const
 ReadBufferFromRemoteFSGather::ReadBufferFromRemoteFSGather(
     const std::string & common_path_prefix_,
     const BlobsPathToSize & blobs_to_read_,
-    const ReadSettings & settings_)
+    const ReadSettings & settings_,
+    bool use_external_buffer_)
     : ReadBuffer(nullptr, 0)
     , common_path_prefix(common_path_prefix_)
     , blobs_to_read(blobs_to_read_)
     , settings(settings_)
+    , use_external_buffer(use_external_buffer_)
```

Comment:
This variable is not used anywhere in this file, let's just remove it.

Reply:
Agreed. I'll check the derived class hierarchy and remove it from  ReadBufferFromRemoteFSGather class.